ui/scripts/jsClientDemo.js

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env node
+
+/** jsClientDemo.js is meant to be used for debugging SQL requests and responses.
+ * You can run it from the command line or using node.
+ *
+ *   $ node jsClientDemo.js
+ *   $ ./jsClientDemo.js
+ *
+ * Once running, you can type in SQL commands and the client will execute them against the default local cockroach
+ * instance.
+ *
+ *   > CREATE DATABASE JS;
+ *   ...
+ *   > SHOW DATABASES;
+ *   ...
+ *
+ *   To exit, press ctrl-C
+ */
+
+var ProtoBuf = require("protobufjs");
+var ByteBuffer = ProtoBuf.ByteBuffer;
+var path = require("path");
+var u = require("util");
+
+var root = ProtoBuf.loadJsonFile(path.join(__dirname, "../generated/wireproto.json")).build(),
+    SQL = root.cockroach.sql.driver;
+
+var request = require("request");
+
+var os = require("os");
+var hostname = os.hostname();
+var addr = "http://" + hostname + ":26257"; // current cockroach default
```

Comment:
I was actually referring to the hostname, because I feel i't confusing that it isn't localhost. I'll move the comment up a line.